mmv1/templates/terraform/examples/tpu_v2_vm_v5p.tf.erb

```diff
@@ -0,0 +1,21 @@
+resource "google_tpu_v2_vm" "<%= ctx[:primary_resource_id] %>" {
+  provider = google-beta
+
+  name = "<%= ctx[:vars]["vm_name"] %>"
+  zone = "us-east5-a"
+
+  runtime_version = "tpu-vm-tf-2.16.1-pjrt"
+
+  accelerator_config {
+    type     = "V5P"
+    topology = "2x2x1"
+  }
+
+  network_config {
+    enable_external_ips = true
```

Comment:
This configuration is essentially unnecessary, but I added it to work around the constraints of the VPC network when running tests.